### DISCUSSION TITLE
zh/faq/

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#23] [Announcements]
# zh/faq/

AList v3的文档

https://alist.nn.ci/zh/faq/

**Comment by jing5460** [comment]
V3 版本音频无法预览播放，请问哪里出了问题

**Reply by jing5460** [reply]
提示：System error: TypeError: Cannot read properties of undefined (reading 'search')

**Reply by jing5460** [reply]
系统:CentOS 7.9.2009 x86_64(Py3.7.9)
V2 版本 音频正常  
 v3  任何格式音频都 
提示：System error: TypeError: Cannot read properties of undefined (reading 'search')  
请求解决

**Reply by 9390041** [reply]
+!

**Comment by piaohong1978** [comment]
V3版本,查看汉字文本文件显示乱码?哪里可以指定汉字编码

**Comment by zzerosama** [comment]
群晖怎么挂载本地存储？没有输账号密码的地方，总是提示不被允许，想挂载本地利用其他移动端设备从NAS里复制文件到其他网盘里

**Reply by liujixu1984** [reply]
挂webdav嘛

**Reply by zzerosama** [reply]
找到挂本地的方法了，挂WebDAV会在移动设备上走一圈数据，数据会先从nas上下载，再传其他网盘，我要的是直接控制nas上传的其他网盘，可发现挂本地也是这样，难受，应该就是Alist本身的传输问题了

**Comment by hhyasl** [comment]
怎么让v3 支持 wma播放音频

**Comment by frozenleaves** [comment]
alist 在潘多拉固件的路由器上运行报Segmentation fault怎么解决啊?内存还有100M剩余的

**Comment by yifan792** [comment]
有没有可能左侧加个目录树？这样多盘操作会方便很多

**Comment by shenlan823** [comment]
运行“./alist server”报错：failed to start: listen tcp 0.0.0.0:5244: bind: address already in use 
这个咋解决。

**Reply by Surely0402** [reply]
你5244端口是不是被占用着的? 你netstat -lntp|grep 5244 看下

**Reply by oldyouth1** [reply]
FATA[2023-01-17 20:55:20] failed to start: listen tcp 0.0.0.0:5244: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

**Comment by wuliujia** [comment]
能在说明里说一下关于内置aria2下载相关问题么？前端后端都按照说明设置了，可是却无法利用alist内的aria2下载项，提示Upload not supported，是什么问题哈

**Comment by oldbig** [comment]
通过alist提供的服务获得的百度网盘的refresh_token, 如何能收回/失效这个refresh_token?
还是有些担心refresh_token 泄露。

**Comment by 184938964** [comment]
哪位大佬知道alist怎么登陆啊，找不到登录入口

**Comment by YzMzx** [comment]
恢复备份卡在设置恢复成功

**Comment by wuciye** [comment]
用aria2离线下载 下载路径是在alist所在路径而不是在aria2设置的下载路径，请问这个可以改吗？

**Reply by xinxinenjoy** [reply]
貌似除了直接调用aria2的时候会放到它自己设置的路径里，其他任何方式调用的都会按照各自的设置去存

**Comment by edgywegj** [comment]
显示“程序“alist.exe”无法运行: 指定的可执行文件不是此操作系统平台的有效应用程序。所在位置 行:1 字符: 1”
我下载的是Windows AMD64包，用了cmd和powershell都是这样。
操作系统：Windows 8.1 专业版 x64

**Comment by IMXeno123** [comment]
服务启时遇到错误：failed to start: listen tcp 0.0.0.0:5244: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
已检查防火墙规则都为允许，检测端口也无其他程序占用。

**Reply by oldyouth1** [reply]
FATA[2023-01-17 20:55:20] failed to start: listen tcp 0.0.0.0:5244: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

**Comment by SourBean** [comment]
V3版本，本地磁盘的PDF文件无法预览，PDF.js提示Failed to fetch，F12查看提示the contet must be served over HTTPS。
Onedrive的可以访问，但网速太卡的时候同样报错。
希望可以解决或者不强制要求https，另外也希望可以加入保存PDF阅读进度。

**Comment by NextEmby** [comment]
bearstar@Macbook alist % ./alist server
INFO[2023-01-03 16:21:50] reading config file: data/config.json        
INFO[2023-01-03 16:21:50] load config from env with prefix: ALIST_     
INFO[2023-01-03 16:21:50] init logrus...                               
WARN[2023-01-03 16:21:50] not enable search                            
INFO[2023-01-03 16:21:50] start server @ 0.0.0.0:5244                  
INFO[2023-01-03 16:21:50] Aria2 not ready.                             
./alist admin

第四条报错 不知道怎么回事儿

**Reply by NextEmby** [reply]
WARN[2023-01-03 16:21:50] not enable search 第四条一直报错

**Comment by hj33032** [comment]
V3.8存储利用/显示根目录排序无效，始终在最地下，这有解决方法吗？

**Comment by limstranger** [comment]
点击主页无法进入百度网盘，显示System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat，请问这种情况该如何解决？

**Reply by Shendai67** [reply]
把油猴扩展关了

**Comment by njuxcy** [comment]
请教各位大神，如何在不同网盘之间同步包括文件夹的文件

**Reply by xinxinenjoy** [reply]
不同网盘间的同步就不要想了，这个只能让你访问的时候方便，网盘间的传输是要“从A下载-存到本地-上传到B”这样的

**Comment by oldyouth1** [comment]
FATA[2023-01-17 20:55:20] failed to start: listen tcp 0.0.0.0:5244: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

**Comment by Godfathery** [comment]
搜索栏，和设置里的索引，点进去都提示Search not available，这个咋解决啊？

**Comment by cey434** [comment]
通过网页版使用IDM下载，为什么下载不全，有的只有149kb,哪里设置有问题吗？

**Reply by cey434** [reply]
请教各位大神

**Comment by darkelfzdy** [comment]
你好我刷新阿里云盘的令牌，用阿里云盘app扫描二维码没有任何反应，请问如何处理？

**Comment by pipinian** [comment]
对拷不能区分失败和成功的任务，失败的任务也不能重试，那么多文件只能人工一个个的对比，真的很不方便，以后会优化吗？

**Comment by zhaoxiaopeng3** [comment]
iStoreOS 安装Alist，启动后一直显示“Alist未运行”，日志里是空白的。请问这种问题怎么解决，我反复安装很多次，改默认端口也不行，谢谢大家帮忙！

**Comment by Shendai67** [comment]
有油猴扩展的时候打开alist显示 System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat，把油猴关了就好了

**Comment by lrs95** [comment]
点击播放器下面的在potplayer里打开，播放错误，网页播放也经常黑屏只有声音

**Comment by guaijie1995** [comment]
alist如何导出索引列表

**Comment by uranus-bb** [comment]
挂载后打开出现failed get objs: failed to list objs: Post "http://alist.xiaoya.pro/api/fs/list": dial tcp: lookup alist.xiaoya.pro on [**1]:53: server misbehaving  该怎么解决

**Comment by vioycq** [comment]
阿里云盘文件夹重命名以后，在alist就访问不了文件了，显示 failed get link: invalid X-Device-Id

**Reply by Huanxige52** [reply]
我目前所有的 阿里 都显示这个  failed get link: invalid X-Device-Id   不知道怎么解决

**Reply by langman002** [reply]
我的也是类似问题

**Comment by Timber552** [comment]
求助，阿里云盘的Token 过期后更新token，一直永不了，软路由上的alist 挂的阿里云盘虽然显示出所以有内容，但是打开（视频）一直是显示加载失败，Infuse添加这个网盘也是没办法播放内容，提示：发生错误，加载此内容发生错误。
哪怕想再手机扫码获取Token，现在简介网页获取token的二维码，一直出不来，一直显示“等待....”

有没有大神可以帮帮指导小弟解决一下。

**Reply by uranus-bb** [reply]
这是阿里云官方封杀了第三方，现在目前自己的网盘不能在线看了，可以看分享的

**Comment by 951337178** [comment]
错误
invalid X-Device-Id
这是哪里出了问题，弄了好几遍还是不行

**Reply by Huanxige52** [reply]
我的阿里云挂载 也是这个问题 不知道咋回事 我重新挂载也不行

**Comment by Huanxige52** [comment]
v2  v3 版本 所有的阿里云盘挂载 失效 提示failed get link: invalid X-Device-Id  请问是哪里出了问题？怎么解决

**Reply by Hanzo-Huang** [reply]
我也这个情况，咋办

**Comment by xingna97** [comment]
各位大神，出现以下问题，如果何解决：获取设置失败： 请稍后，正在加载储存。
之前一直使用正常，电脑重启后出现这个问题 。不知为何？

**Comment by langman002** [comment]
我的alist阿里云盘提示:failed get objs: failed to list objs: invalid X-Device-Id
是怎么回事，换了token也不行。

**Reply by Hanzo-Huang** [reply]
我下载最新的alist放进去就可以了

**Comment by chaes666** [comment]
遇到个问题,admin账号的密码不对,使用alist admin命令查看了密码,再次输入始终不对,是怎么回事.清理掉缓存文件一样这个问题.

**Comment by ShuaiKangChen** [comment]
在浏览器中使用streamsaver而不是服务器来进行包下载需要相应的存储支持cors，而不支持的存储将失败

**Comment by xx6z8v** [comment]
执行./alist server总是报错exit status 126,我并不了解是少了什么权限和解决方法

**Comment by ViaVxxx** [comment]
使用阿里云盘open里的Aliyundrive Refresh Token Callback获取的刷新令牌，填入保存时提示 Failed init storage: failed to refresh token: The input parameter refresh_token is not valid.
刷新令牌无效

**Comment by Rpower666** [comment]
阿里云盘 open出现failed get objs: failed to list objs: Post "https://api.nn.ci/alist/ali_open/token": EOF，不知什么原因

**Comment by lanhuakai** [comment]
安装遇到这个问题，有朋友遇到吗？
C:\Users\AOAO\Downloads\Compressed\alist-windows-amd64>alist.exe server
INFO[2023-03-05 11:58:23] reading config file: data\config.json
INFO[2023-03-05 11:58:24] load config from env with prefix: ALIST_
INFO[2023-03-05 11:58:24] init logrus...
INFO[2023-03-05 11:58:24] start server @ 0.0.0.0:5244
INFO[2023-03-05 11:58:28] Aria2 not ready.
INFO[2023-03-05 11:58:28] qbittorrent not ready.

**Comment by langman002** [comment]
config file: data\config.json



在 2023-03-05 12:07:08，"lanhuakai" ***@***.***> 写道：

安装遇到这个问题，有朋友遇到吗？
C:\Users\AOAO\Downloads\Compressed\alist-windows-amd64>alist.exe server
INFO[2023-03-05 11:58:23] reading config file: data\config.json
INFO[2023-03-05 11:58:24] load config from env with prefix: ALIST_
INFO[2023-03-05 11:58:24] init logrus...
INFO[2023-03-05 11:58:24] start server @ 0.0.0.0:5244
INFO[2023-03-05 11:58:28] Aria2 not ready.
INFO[2023-03-05 11:58:28] qbittorrent not ready.

**Comment by langman002** [comment]
config file: data\config.json
配置文件：数据\配置.json
是不是config.json文件里的alist目录地址路径和你安装的alist文件路径不匹配呀。你查看一下。



在 2023-03-05 12:07:08，"lanhuakai" ***@***.***> 写道：

安装遇到这个问题，有朋友遇到吗？
C:\Users\AOAO\Downloads\Compressed\alist-windows-amd64>alist.exe server
INFO[2023-03-05 11:58:23] reading config file: data\config.json
INFO[2023-03-05 11:58:24] load config from env with prefix: ALIST_
INFO[2023-03-05 11:58:24] init logrus...
INFO[2023-03-05 11:58:24] start server @ 0.0.0.0:5244
INFO[2023-03-05 11:58:28] Aria2 not ready.
INFO[2023-03-05 11:58:28] qbittorrent not ready.

**Comment by langman002** [comment]
阿里云网盘屏蔽了alist。获取不了阿里云的token.或者是显示的token都失效了，根本不能用。所以会出现获取项目列表失效，导致获取失败、希望阿里云能大度一些。解除这些限制。



在 2023-03-02 17:16:34，"Rpower666" ***@***.***> 写道：

阿里云盘 open出现failed get objs: failed to list objs: Post "https://api.nn.ci/alist/ali_open/token": EOF，不知什么原因

**Comment by zhangfaji** [comment]
在线预览 能否用自己部署的 kkFileView？

**Comment by JerryLiang-go** [comment]
小米路由ax6000，基于openwrt的系统，可ssh，能安装吗....

**Comment by lightning110** [comment]
服务器宝塔安装v3 ，webdav报错：Method Not Allowed

**Reply by lightning110** [reply]
有大佬告知一下是什么原因吗

**Comment by masterchen2** [comment]
root@aml-s812:~# ./alist server
INFO[2023-03-13 20:36:18] reading config file: data/config.json        
INFO[2023-03-13 20:36:18] config file not exists, creating default config file 
INFO[2023-03-13 20:36:18] load config from env with prefix: ALIST_     
INFO[2023-03-13 20:36:18] init logrus...                               
INFO[2023-03-13 20:36:18] Successfully created the admin user and the initial password is: XevbkErd 
INFO[2023-03-13 20:36:18] Aria2 not ready.                             
INFO[2023-03-13 20:36:18] start server @ 0.0.0.0:5244                  
INFO[2023-03-13 20:36:18] qbittorrent not ready.

**Reply by masterchen2** [reply]
一直卡在这个阶段

**Reply by wangzikang0728** [reply]
我也是同样的问题，请问有解决吗？我是安卓的termux上出现的

**Comment by FCQ1400** [comment]
求大神指点：在alist挂在百度网盘后，使用raidrive时，在出现的文件上有X标志，并且只能下载文件不上上传文件，这个问题怎么解决？

**Comment by masterchen2** [comment]
百度网盘的问题



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年03月18日 11:19 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [alist-org/docs] zh/faq/ (Discussion #23) |

求大神指点：在alist挂在百度网盘后，使用raidrive时，在出现的文件上有X标志，并且只能下载文件不上上传文件，这个问题怎么解决？

**Comment by wtaotao2019** [comment]
挂载阿里网盘，web端可以预览文件，群晖挂载WebDAV，地址 端口 路径 账号密码都是对，连接就提示 连接被服务拒绝，是哪里出问题

**Comment by Xtnayue** [comment]
请问Android系统使用termux启动alist后，进行备份，其备份出的文件在哪里

**Comment by x452468405** [comment]
之前用clouddrive能播放  电脑重启后 就不能 播视频 后面下了 alist什么都弄好了 也是视频播放不了 文件下载不了啥问题 是不是电脑什么配置 自动改了？

**Comment by wanghuia101** [comment]
Raidrive映射alist后上传文件失败,显示“您需要权限来执行此操作”是哪个地方的问题?
我用阿里云盘,没有出现这样的问题

**Comment by WUFXXX** [comment]
win桌面端自动更新了，115只能浏览文件列表无法访问文件，无法播放视频下载文件，手机端旧版正常使用，升级3.14后同上无法访问115，希望尽快发布稳定版吧

**Comment by weiyouai1** [comment]
速度可以，但是用KODI没办法用，刮削以后在电影或者电视剧海报墙那里就不能播放像掉盘一样，只能在视频位置能播放，用群晖安装Alist或者软路由都一样，在群晖用clouddrive可以挂载为本地或者软路由安装阿里云盘WedDav都能正常使用。

**Comment by dranshion** [comment]
alist 迅雷专家版 代理方式选302是不是无效的？   还是相当于本地代理！！！

**Comment by lanzby** [comment]
添加阿里云盘时候提示I400JD:JWT Deserialize Failed: `Jose failed `Bearer``，什么意思，用不了

**Comment by xq-soliton** [comment]
123云盘视频没有一个播放器能正常播放，请求修复一下

**Comment by yeluonight** [comment]
115好像用不了了，我之前是在路由器和电脑都用alist挂载的同一个115账户，可以同时在线，但是现在只能登陆一个，登第二个alist就会把第一个的挤掉，试过使用同一token、同一cookie都不行，每个alist用不同的也不行，尝试抓取115手机客户端的cookie反而导致115客户端掉线（以前是可以的）
（）

**Comment by zhangzp999** [comment]
能让浏览器直接打开一个alist挂载中的html进行浏览吗？现在浏览器一打开地址就会自动下载 .而安卓手机微信环境是不能自动下载的

**Comment by fanxu1218** [comment]
路由器版本 V1.6.8 播放不了115网盘

**Comment by jinbulee** [comment]
docker 启动的，把/opt/alist/data/挂在到服务器本地的/data/alist/ 目录下面，今天服务器断电重启之后，/data/alist/temp 下面的文件全没了。是哪里设置的问题吗。是不是找不回来了

**Comment by magicvdog** [comment]
linux 无法挂载本地目录 居然怎么搞。

**Comment by Jerry12228** [comment]
前端下载的"发送到aria2"不能下载整个文件夹吗?

**Comment by fanxu1218** [comment]
目前alist官方二进制v3.17.0，华硕路由器挂载，依然无法读取115网盘的资源，麻烦作者大大解决一下

**Comment by letuplus** [comment]
alist 版本 3.18 ，在网页播放 wma 文件提示 an audio error has occurred，但复制url后，在potplayer 和 Windows media player 播放均正常。

**Comment by tianchaopow** [comment]
接口令牌一直处于刷新状态，无法获取refresh_token:

**Comment by PINGXcpost** [comment]
搭建好了，我是onedrive云盘，如果我下载100Gb的东西，会消耗我服务器100Gb流量吗

**Comment by meitu007** [comment]
123云盘错误 Failed init storage: 123pan forbidden，怎么解决

**Comment by xq-soliton** [comment]
123云盘不支持播放了，不对外开放
        
        
            
        
    
         
        
            
                        ***@***.***
            
        
     
        
  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            ***@***.***>
            
        
      
      
        
         Date 
        
        
        7/17/2023 09:42
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              Re: [alist-org/docs] zh/faq/ (Discussion #23)
        
      
    
  

123云盘错误 Failed init storage: 123pan forbidden，怎么解决

**Comment by chenpeng198877** [comment]
挂载对象储存b2  所有文件夹无法获取上传时间和大小，所有文件夹都记录最后刷新时间！希望能尽快修复一下，资源较多无时间记录很难管理！

**Comment by frank916528** [comment]
ikuai怎么部署alist，求教！！！

**Comment by xq-soliton** [comment]
看看hosts主机映射文件内容<br/><br/><br/><html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body>
<div class="ntes-mailmaster-quote" style="padding-top: 1px; padding-bottom: 1px" >
    <div style=" margin-top: 20px; margin-bottom: 12px; font-size: 14px; line-height: 1.25; color: #89898c; " >---- 回复的原邮件 ----</div>
    <div style=" margin-bottom: 12px; font-size: 13px; line-height: 1.25; color: #2c2c2e; padding: 2px; border-radius: 8px; background-color: #f0f0f0; " >
        <table width="100%" cellpadding="0" cellspacing="10" border="0">
            <tr>
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; white-space: nowrap; " >发件人</td>
                <td valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " ><a class="mail-from" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a></td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; white-space: nowrap; " >日期</td>
                <td class="mail-date" valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " >2023年09月21日 08:56</td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; " >收件人</td>
                <td valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " ><a class="mail-to" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a></td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; " >抄送至</td>
                <td valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " ><a class="mail-cc" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a>、<a class="mail-cc" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a></td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; " >主题</td>
                <td class="mail-subject" valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " >Re: [alist-org/docs] zh/faq/ (Discussion #23)</td>
            </tr>
        </table>
    </div>
    <div><p></p>
<p dir="auto">请问一下，现在电脑当主机可以访问了，其他手机IPAD不知道为啥一直访问不了127.0.0.1不行，电脑IP也不行</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br />Reply to this email directly, <a href="https://github.com/alist-org/docs/discussions/23#discussioncomment-7064157">view it on GitHub</a>, or <a href="https://github.com/notifications/unsubscribe-auth/AKRELNCH5ZWJYS7AHNW5733X3OGCZANCNFSM6AAAAAAQTVDHHM">unsubscribe</a>.<br />You are receiving this because you commented.<img src="https://github.com/notifications/beacon/AKRELNENM227ZDTP65QA5X3X3OGCZA5CNFSM6AAAAAAQTVDHHOWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQANPFF2.gif" height="1" width="1" alt="" /><span style="color: transparent; font-size: 0; display: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; height: 0; max-width: 0; max-height: 0; mso-hide: all">Message ID: <span>&lt;alist-org/docs/repo-discussions/23/comments/7064157</span><span>@</span><span>github</span><span>.</span><span>com&gt;</span></span></p>
<script type="application/ld+json">[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/alist-org/docs/discussions/23#discussioncomment-7064157",
"url": "https://github.com/alist-org/docs/discussions/23#discussioncomment-7064157",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]</script></div>
</div>
</body>
</html>

**Comment by issacharry** [comment]
為什麼我用小雅的時候明明正在看A電影，但卻幫我轉存A，B，C....的電影， 弄得經常自己的盤爆滿

**Comment by oooorange12138** [comment]
win11 打不开127.0.0.1:5244 有大佬知道怎么弄吗

**Comment by Seventeenslove** [comment]
error：
[del_user_cache_online] failed: Post "http://localhost:5244/api/admin/user/del_cache?username=admin": dial tcp [::1]:5244: connect: connection refused
help

**Comment by sntu11** [comment]
alist第三方播放时，点potplayer可以使用，但是VLC无响应。调不出来。

**Comment by Longinuspeng** [comment]
上百度秒传的链接，通过链接转换可以直接出来哈希值，是不是理论上可以秒传到天翼？

**Comment by blw108** [comment]
曲奇云盘还支持吗，配置页面找不到，但是看说明文档里是有的

**Comment by DreamCating** [comment]
您好，请问windows客户端打开白屏要如何解决，从历史记录的（alist管理页面能从历史记录进去）进去也是白屏包括管理页面，并且整个页面白屏没有能进管理页面的入口，重新安装和配置存储后还是如此

**Comment by qq178638** [comment]
百度云盘 下载大型文件出现{"error_code":31362,"error_msg":"sign error","error_info":"","request_id":2160069397306279657}

**Comment by xiaoyi0805** [comment]
alist部署在云服务器上再挂在阿里云盘，访问的速度是看服务器网速还是阿里云盘网速

**Comment by xiaoyi0805** [comment]
alist部署在云服务器上再挂在阿里云盘，访问的速度是看服务器网速还是阿里云盘网速

**Comment by xiaoyi0805** [comment]
把alist部署到云服务器上，服务器网速只有5m。云服务器上的alist挂载阿里云盘，那访问的速度是5m还是阿里云盘webdav的速度

**Comment by TonyLiuDev** [comment]
alist 支持挂载同一类型网盘多账户吗？

**Comment by Lxj96** [comment]
下载链接url的端口没有拼接
复制链接地址没问题，例如：http://xxxxx.top:45245/d/111.jpeg?sign=Ug-pXsx7grgHgXmpnIGAPBh_7rM9Gq83cz1FCfAl3c8=:0
但是下载的链接变成了：http://xxxxx.top/p/111.jpeg?sign=Ug-pXsx7grgHgXmpnIGAPBh_7rM9Gq83cz1FCfAl3c8=:0
查看网络请求，/api/fs/get返回的raw_url就没有端口

**Comment by zhujieqaq** [comment]
有没有方法设置播放下一个视频的时候，倍速不变，我播放下一个视频，1.5就会变成1.0

**Comment by 0xxxxx** [comment]
init tool aria2 failed: failed get aria2 version: Post "http://localhost:6800/jsonrpc": dial tcp 127.0.0.1:6800: connect: connection refused 


这个是啥问题 我安装步骤都是正确的

**Reply by 0xxxxx** [reply]
[del_user_cache_online] failed: Post "http://localhost:5244/api/admin/user/del_cache?username=admin": dial tcp 127.0.0.1:5244: connect: connection refused 
 
设置用户名密码

**Comment by zengyehx** [comment]
电脑端清理缓存后无法登录，登录网页提示：Guest user is disabled, login please。以前可以直接登录，现在点登录按键提示：Too many unsuccessful sign-in attempts have been made using an incorrect username or password, Try again later.
请问大家要怎么操作，我是小白，只知道用电脑端

**Reply by nickdlkk** [reply]
看下有没其它缓存，我反代设置了缓存也导致了这个错误。

**Comment by viaoh** [comment]
用着用着出现这个问题：Request failed with status code 500，所有都打不开了，怎么解决

**Comment by hermano7788** [comment]
Failed init storage: Get "https://passportapi.115.com/app/1.0/web/1.0/check/sso?_=1715822904305": tls: first record does not look like a TLS handshake使用桌面版后所有的网盘都会这样。请问是什么问题，怎么解决·

**Comment by 875272076** [comment]
挂载pikpak挂载不了。老是出现这样的问题， 有没有大神解答一下
Failed init storage: Post
×"https://user.mypikpak.com/v1/auth/signin":read tcp
*****：36870-
83:443: read: connection reset
by peer

**Comment by weituw** [comment]
是用内网穿透提示这个，好像是UI的问题，有什么解决办法吗。System error: Error: [Hope UI]: useColorMode must be used within a HopeProvider

**Comment by mxl101** [comment]
pdf和docx文件看不了怎么回事儿？

**Comment by YannZhou** [comment]
软件显示[AD] Command finished with code 1 and signal null已停止
日志：[33mWARN[0m[2024-06-11 16:18:08] not enable search                            
[GIN] 2024/06/11 - 16:18:49 | 200 |       518.2µs |             ::1 | POST     "/api/admin/user/del_cache?username=admin"
[GIN] 2024/06/11 - 16:19:05 | 200 |      7.0078ms |       127.0.0.1 | GET      "/"
[GIN] 2024/06/11 - 16:19:06 | 200 |        60.5µs |       127.0.0.1 | GET      "/api/public/settings"
[GIN] 2024/06/11 - 16:19:06 | 200 |         681µs |       127.0.0.1 | GET      "/api/me"
[GIN] 2024/06/11 - 16:19:12 | 200 |       103.3µs |       127.0.0.1 | POST     "/api/auth/login/hash"
[GIN] 2024/06/11 - 16:19:12 | 200 |      1.1321ms |       127.0.0.1 | GET      "/api/me"
[GIN] 2024/06/11 - 16:19:12 | 200 |            0s |       127.0.0.1 | GET      "/api/public/offline_download_tools"

**Comment by winining** [comment]
请问使用mac版的Alist desktop软件，想使用同步软件同步电脑上的内容，但是尝试了几个软件，都不能同步，大概提示是没有找新的更新内容，请问是什么原因？需要怎么配置才能实现mac电脑的文件同步到百度云上？感谢。

**Comment by Johy-Smith** [comment]
请问这个错误是什么原因导致的呢？Put "http://fhnfile.cn-shenzhen.oss.aliyuncs.com/66be0ea024822c8dfac3c97a6ac230543cfe8149?partNumber=327&uploadId=3963B1816EA34B8ABD10C166755AC986": net/http: timeout awaiting response headers

**Comment by yc-forever** [comment]
database is locked

**Comment by lizeyu123456789** [comment]
怎么限定用户只能访问某个文件

**Comment by kings12369** [comment]
启动后一直提示
获取设置失败： 请稍候，正在加载储存

**Comment by litian2796** [comment]
升级v3.37.4 版本后   不显示日志，请解决

**Comment by xiaosa70** [comment]
从123盘复制到onedrive盘的时候一部分文件提示这个
{"error":{"code":"invalidRequest","message":"Invalid request"}}
想请教一下大佬怎么解决

**Comment by xia0028** [comment]
有没有大佬知道，alist部署在服务器上在本地电脑RaiDrive上怎么挂载。
3450149878@qq.com希望能发给我
